BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Channel/TransmissionTest.cs

```diff
@@ -363,6 +366,50 @@ public async Task SendAsyncReturnsCorrectHttpResponseWrapperWithRetryHeaderWhenN
                 }
 
             }
+
+            [TestMethod]
+            public async Task SendAsyncLogsBreezeReponseTimeAndStatusCode()
+            {
+                var handler = new HandlerForFakeHttpClient
+                {
+                    InnerHandler = new HttpClientHandler(),
+                    OnSendAsync = (req, cancellationToken) =>
+                    {
+                        // VALIDATE
+                        Assert.AreEqual(testUri, req.RequestUri);
+                        Assert.AreEqual(HttpMethod.Post, req.Method);
+                        return Task.FromResult<HttpResponseMessage>(new HttpResponseMessage());
+                    }
+                };
+
+                using (var fakeHttpClient = new HttpClient(handler))
+                {
+                    var items = new List<ITelemetry> { new EventTelemetry(), new EventTelemetry() };
+
+                    // Instantiate Transmission with the mock HttpClient
+                    Transmission transmission = new Transmission(testUri, new byte[] { 1, 2, 3, 4, 5 }, fakeHttpClient, string.Empty, string.Empty);
+                    // transmission.Timeout = TimeSpan.FromMilliseconds(1);
+
+                    using (var listener = new TestEventListener())
+                    {
+                        var eventCounterArguments = new Dictionary<string, string>
+                        {
+                            {"EventCounterIntervalSec", "1"}
+                        };
+
+                        listener.EnableEvents(CoreEventSource.Log, EventLevel.LogAlways, (EventKeywords)AllKeywords, eventCounterArguments);
+
+                        HttpWebResponseWrapper result = await transmission.SendAsync();
+
+                        // VERIFY
+                        // We validate by checking SDK traces.
+                        var allTraces = listener.Messages.ToList();
+                        // Event 67 is logged after response from breeze.
+                        var traces = allTraces.Where(item => item.EventId == 67).ToList();
```

Comment:
we dont really need test for event67, but need one for the actual counter being written.

Reply:
Introduced this new event 67, will leave this test case to handle breeze time log. I will add a new test case for counter.